tests/integration/test_pairwise_significance.py

```diff
@@ -512,25 +512,29 @@ def test_pairwise_significance_mr_x_mr(self):
     def test_pairwise_cat_x_mr_gender_x_all_pets_owned(self):
```

Comment:
I would use the suffix `_weighted` where it's needed in the test name. This way we understand that we are testing the cube with a weight